tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -681,7 +682,8 @@ func (az *AzureProvider) CreateWindowsVM() (err error) {
 		log.Info(fmt.Sprintf("Please Check for file %s in %s directory on how to access the node",
 			instanceName, az.resourceTrackerDir))
 	}
-	return nil
+	credentials := types.NewCredentials("", *ipAddress, adminPassword)
```

Comment:
I've got a question, why are you passing " " even though we've got instanceName, are you doing because it isn't ID..?

Reply:
Yeah, I wasn't sure about azure, if the instanceName is treated as ID, if yes, I can pass the name itself.

Reply:
Yeah the instanceName would be unique for a particular resourceGroup name, and if you're trying to create an instance with the same name then basically you're trying to update the same instance. So yeah we can treat name as ID in my opinion.

Reply:
Done.